bunsen/bunsen-core/pom.xml

```diff
@@ -38,7 +38,20 @@
       <artifactId>hapi-fhir-structures-dstu3</artifactId>
       <scope>compile</scope>
     </dependency>
-
+    <dependency>
+      <groupId>ca.uhn.hapi.fhir</groupId>
+      <artifactId>hapi-fhir-validation-resources-r4</artifactId>
```

Comment:
Don't we require this and the next dep in the production code too? I mean why just `test` scope? If these are only needed for `test` scope, then for example, should we also update `hapi-fhir-structures-dstu3` above to `test` scope as well?